gateway/install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+type docker &> /dev/null || { echo "Docker is not instaled"; exit 1; }
+
+if !docker info &> /dev/null; then
+    cat <<EOF
+Docker is not running or your current user is not in docker group.
+You need to manually add your current user to docker group or run this installer using sudo.
+
+EOF
+    while true; do
+	echo -n "Do you want to run the installer using sudo? [Y/n] "
```

Comment:
Here it should be: `[y/N]` and `N` should be the default.

Reply:
Eu achei que o sim deveria ser o padrão